plugin/storage/es/options.go

```diff
@@ -119,6 +120,10 @@ func addFlags(flagSet *flag.FlagSet, nsConfig *namespaceConfig) {
 		nsConfig.namespace+suffixPassword,
 		nsConfig.Password,
 		"The password required by ElasticSearch")
+	flagSet.String(
+		nsConfig.namespace+suffixTokenPath,
+		nsConfig.TokenFilePath,
+		"Path to a file containing bearer token. This flag also uses "+suffixCA+" if it is specified")
```

Comment:
`suffixCA` or `suffixTokenPath`? I think it might be implicit, but the token is the whole content of the file, right? Perhaps we could make it more explicit?

Reply:
`suffixCA`, it's not a bad thing to have more explanatory docs. I didn't know that we need root CA at all when using token.

Reply:
I wouldn't use the constant in the help message. It starts with a dot and will look weird. Just say CA.